gel/_testbase.py

```diff
@@ -623,6 +630,54 @@ def adapt_call(cls, result):
         return result
 
 
+class SQLATestCase(SyncQueryTestCase):
+    SQLAPACKAGE = None
+    DEFAULT_MODULE = 'default'
+
+    @classmethod
+    def setUpClass(cls):
+        # SQLAlchemy relies on psycopg2 to connect to Postgres and thus we
+        # need it to run tests. Unfortunately not all test environemnts might
+        # have psycopg2 installed, as long as we run this in the test
+        # environments that have this, it is fine since we're not expecting
+        # different functionality based on flavours of psycopg2.
+        if importlib.util.find_spec("psycopg2") is None:
```

Comment:
I think it would be cleaner to do 
```
try:
    import psychopg2
except ImportError:
    # handle it
```

Reply:
I wanted to avoid the import since I don't even _need_ anything from `psycopg2` directly, not using it anywhere myself. Didn't want to leave the impression that I needed the import myself and if it's not used, maybe it's OK to "remove this dependency check" making it look like an outdated comment or dead code. `importlib` seemed like a more overt and unambiguous check.